calculator.rb

```diff
@@ -0,0 +1,86 @@
+
+
+# baseline: the program should ask for an operation (string or numeric symbol) and two numbers
+puts "Welcome to Anne's CLI-culator!"
+
+num_a = 0
+num_b = 0
+
+calculator_functions = ["add or +", "subtract or -", "multiply or *", "divide or /"] # array to print out the functionality currently supported
+puts "Please choose from the following operations: "
+puts calculator_functions
+print "What would you like to do: "
+command = gets.chomp.downcase
+
+puts "Great! Let's #{command} two numbers!"
+puts "Please enter the first number to #{command}: "
+begin #should maybe make this rescue into a method so it can be used on num_b
+  num_a = gets.chomp
+  num_a = Integer(num_a)
+rescue
+  print "Please enter an integer: "
+  retry
+end
+
+puts "Please enter the second number to #{command}: "
+
+begin
+  num_b = gets.chomp
+  num_b = Integer(num_b)
+rescue
+  print "Please enter an integer: "
+  retry
```

Comment:
Neat work using a `rescue` and `retry.`  Notice that you do this a couple of times... It could be refactored into a method.